gotestcover.go

```diff
@@ -33,6 +33,9 @@ var (
 
 	// custom
 	flagParallelPackages = runtime.GOMAXPROCS(0)
+
+	// GAE/Go
+	flagGoAppEngine bool
```

Comment:
I think we should remove the "Go" prefix: it's redundant (gotestcover only works with Go).
And the official name is "Google App Engine".
I suggest to change the variable name to flagGoogleAppEngine.